Plots/Scatter/NCL_scatter_3.py

```diff
@@ -0,0 +1,98 @@
+"""
+NCL_scatter_3.py
+================
+This script illustrates the following concepts:
+    - Drawing a scatter plot over a map
+    - Drawing markers on a map
+    - Attaching markers to a map
+    - Changing the marker color and size in a map plot
+    - Drawing markers on a map indicating the locations of station data
+   
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/scatter_3.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/scatter_3_1_lg.png and https://www.ncl.ucar.edu/Applications/Images/scatter_3_2_lg.png
+                         
+"""
+
+###############################################################################
+# Import packages:
+
+import xarray as xr
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+from cartopy.mpl.gridliner import LongitudeFormatter, LatitudeFormatter
+import matplotlib.pyplot as plt
+
+from geocat.viz import util as gvutil
+import geocat.datafiles as gdf
+
+###############################################################################
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+
+ds = xr.open_dataset(gdf.get("netcdf_files/95031800_sao.cdf"), decode_times=False)
+lat = ds.lat.isel()
+lon = ds.lon.isel()
+
+###############################################################################
+# Plot
+
+def Plots(xext, yext, xtic, ytic, xminor, yminor, size, color):
```

Comment:
Totally optional, but I think changing xext/yext to xlim and ylim might be good for readability and make more sense (like matplotlib's pyplot xlim/ylim parameters)